src/hotspot/share/runtime/jniHandles.inline.hpp

```diff
@@ -97,8 +118,9 @@ inline oop JNIHandles::resolve_non_null(jobject handle) {
 
 inline void JNIHandles::destroy_local(jobject handle) {
   if (handle != NULL) {
-    assert(!is_jweak(handle), "Invalid JNI local handle");
-    NativeAccess<>::oop_store(jobject_ptr(handle), (oop)NULL);
+    assert(!is_jweak_tagged(handle), "Invalid JNI local handle");
+    assert(!is_global_tagged(handle), "Invalid JNI local handle");
```

Comment:
Why isn't `is_local_tagged` used here?

Reply:
Yeah I just missed it. I see there are more asserts which should just be one tag check. Will fix.